bitcoin/src/blockdata/witness.rs

```diff
@@ -151,29 +151,30 @@ internal_macros::define_extension_trait! {
             self.push(signature.serialize())
         }
 
-        /// Get Tapscript following BIP341 rules regarding accounting for an annex.
+        /// If this witness is a Taproot script spend, return the leaf script.
```

Comment:
Perhaps we should use the 'functions docs use plural' thing that we started. Clearly 

```rust
/// Gets the leaf script if this witness is a Taproot script spend.
```

Note 

```bash
gg '/// Returns' | wc -l
367
gg '/// Gets' | wc -l
24
```